CefSharp/Internals/BrowserProcessServiceHost.cs

```diff
@@ -52,13 +52,15 @@ public void SetOperationContext(OperationContext operationContext)
         public Task<JavascriptResponse> EvaluateScriptAsync(int browserId, long frameId, string script, TimeSpan? timeout)
         {
             var operationContextTask = operationContextTaskCompletionSource.Task;
+            // Use TaskScheduler.Default to ensure that the work occurs
+            // on a background thread.
             return operationContextTask.ContinueWith(t =>
             {
                 var context = t.Result;
                 var renderProcess = context.GetCallbackChannel<IRenderProcess>();
                 var asyncResult = renderProcess.BeginEvaluateScriptAsync(browserId, frameId, script, timeout, null, null);
                 return Task.Factory.FromAsync<JavascriptResponse>(asyncResult, renderProcess.EndEvaluateScriptAsync);
-            }).Unwrap();
+            }, TaskScheduler.Default).Unwrap();
```

Comment:
Would have been ideal as a separate commit, oh well :smile: 

Makes sense to execute on the `ThreadPool`. I think adding something to the documentation would also be worthwhile.